service/resources/records.py

```diff
@@ -0,0 +1,28 @@
+"""Records module"""
+import json
+import falcon
+import jsend
+from ..models.record import Record
+
+class Records():
+    """ Records class """
+    def on_get(self, req, resp, parcel):
+        """ on get request
+        return Records data
+        """
+        if parcel:
```

Comment:
what's a parcel? is the variable a parcel number/id? Also a bit curious what 'records' is because I haven't seen this code before

Reply:
Closing I found so far to explain a parcel is  https://www.redfin.com/definition/apn

I am just using `records` as a generic term. 

61707cc4dccc6aa28d213a734d998e80888d2ea8